integration

```diff
@@ -66,6 +66,8 @@
   "edenhaus/ha-prosenic",
   "eifinger/open_route_service",
   "elad-bar/ha-blueiris",
+  "elad-bar/ha-dahuavto",
+  "elad-bar/ha-hpprinter",
```

Comment:
Only add one change pr. PR, and only manage your own repositories.

Reply:
Fixed.